src/osd/OpRequest.cc

```diff
@@ -93,6 +93,13 @@ void OpRequest::_dump_op_descriptor_unlocked(ostream& stream) const
   get_req()->print(stream);
 }
 
+uint64_t OpRequest::get_poolid() const
+{
+  auto m = dynamic_cast<const MOSDFastDispatchOp*>(get_req());
```

Comment:
should add the relevant #include to this file

Reply:
actually - see my previous comment. It shouldn't be here.

Reply:
Yes, removed it from OpRequest and fetching poolid directly in slow request handling code.